ui/helpers/utils/permission.js

```diff
@@ -293,6 +292,7 @@ export const PERMISSION_DESCRIPTIONS = deepFreeze({
         ...baseDescription,
         label: t('permission_accessSnap', [snapId]),
         description: t('permission_accessSnapDescription', [snapId]),
+        weight: PermissionWeight.wallet_snap,
```

Comment:
Any reason not to add this to the `baseDescription`?

Reply:
No reason. This was my mistake. Fixed now 👍